src/plugins/foxPage/components/OtherOpportunities/FoxOtherOpportunityPanelRow.tsx

```diff
@@ -115,10 +143,7 @@ export const FoxOtherOpportunityPanelRow: React.FC<FoxOtherOpportunityPanelRowPr
           {opportunity.title}
         </CText>
       </Flex>
-      <Skeleton
-        isLoaded={opportunity.isLoaded ? true : false}
-        textAlign={{ base: 'right', md: 'center' }}
-      >
+      <Skeleton isLoaded={Boolean(earnOpportunity)} textAlign={{ base: 'right', md: 'center' }}>
```

Comment:
Now that we properly have selectors returning undefined, that's our free loading state